lesson2/squaring-while-1000.py

```diff
@@ -0,0 +1,7 @@
+if __name__ == "__main__":
+    y = 0
+    while True:
```

Comment:
Try to avoid while True.
```suggestion
    while y**2 < 1000:
```